### DISCUSSION TITLE
blog/zoom-obs-bilibili-broadcast/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#323] [Announcements]
# blog/zoom-obs-bilibili-broadcast/

本文将指导你如何配置和使用 OBS + zoom在 Bilibili 上直播。

https://jimmysong.io/blog/zoom-obs-bilibili-broadcast/

**Comment by itbj** [comment]
这里又遇到了。真巧。我应该有您的微信。推荐你用用思科Webex 做个直播。谢谢。

**Reply by rootsongjc** [reply]
感谢你的建议，Webex 我不是很熟悉。现在云原生社区直接使用腾讯会议主持分享，同时也支持 RTMP 格式的推流，一个软件解决会议和直播问题，可能更方便一些。
